docker/startup.sh

```diff
@@ -15,7 +15,8 @@ sed -i "s/!SECRET_KEY!/$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | head -c 80)/g"
 
 # Refresh the environmental config for DB and Solr hosts in case of IP changes
 cp /etc/openoni.ini.orig /etc/openoni.ini
-sed -i "s/!DB_HOST!/mysql/g" /etc/openoni.ini
+
+sed -i "s/!DB_HOST!/rdbms/g" /etc/openoni.ini
```

Comment:
I don't think you'll need the sed commands for DB_HOST or SOLR_HOST anymore, since you fixed it in the INI file